### ISSUE TITLE
Transposition Cipher Problem

### ISSUE THREAD

**Issue by DolphinLHT** [jbainbridge98/cipherApp#4]
![bug1](https://user-images.githubusercontent.com/56650412/144775645-a04a5f5d-883d-487d-95b7-79623100b634.png)

**Comment by jbainbridge98** [comment]
Yes, another issue with the global variable.

**Comment by jbainbridge98** [comment]
Someone has already found this bug unfortunately

**Comment by symxmyz233** [comment]
Yes, someone has found it ahead of this issue. **It is not a but here.**
